rust/feature-flags/src/team.rs

```diff
@@ -37,6 +37,30 @@ impl Team {
         Ok(team)
     }
 
+    #[instrument(skip_all)]
+    pub async fn update_redis_cache(
```

Comment:
I use this to update redis if lookups fail, see here: https://github.com/PostHog/posthog/pull/24089/files#diff-c0297f6a8e8987e4ba468c0b3c2a41ecaa4c85a541d4f3758e5a27752a64494aR68-R83